mlos_bench/mlos_bench/config/global/env-azure-ubuntu-redis.jsonc

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Azure VM Ubuntu Redis",
+    "class": "mlos_bench.environment.CompositeEnv",
+
+    "include_tunables": [
+        "global/tunables.jsonc"
+    ],
+
+    "include_services": [
+        "services/remote/azure/service-provision-vm.jsonc",
+        "services/remote/azure/service-fileshare.jsonc"
+    ],
+
+    "config": {
+        "include_children": [
+            "environment/vm/azure/provision-ubuntu-vm.jsonc",
+            "environment/os/linux/boot/boot-ubuntu.jsonc",
+            "environment/os/linux/runtime/setup-ubuntu.jsonc",
+            "environment/apps/redis/redis.jsonc"
```

Comment:
Something I'm not sure I like about this layout or not: how verbose these paths are now.
It might be nice if service loading knew it needed to look in the "services/" subpath, where as environment loading knew it needed to look in the "environment/" subdir, but that's a more substantial change.